.github/workflows/reproduce_paper.yml

```diff
@@ -0,0 +1,64 @@
+name: Reproduce Paper
+# This action runs a Jupyter notebook in the examples folder that uses astartes to generate
+# train/val/test splits on two datasets (RDB7 and QM9). These dynamically generated splits
+# are then compared to the splits which were created at the time of paper writing, to ensure
+# that subsequent releases of astartes have backwards compatibility.
+on:
+  schedule:
+    - cron: "0 8 * * 1-5"
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+
+concurrency:
+  group: actions-id-${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -el {0}
+    name: Reproduce Paper Data Splits
+    steps:
+      - uses: actions/checkout@v3
+      - uses: mamba-org/provision-with-micromamba@main
+        with:
+          environment-file: false
+          environment-name: temp
+          channels: defaults,conda-forge
+          channel-priority: flexible
+          extra-specs: |
+            python=3.11
+      - name: Install Dependencies
+        run: |
+          python -m pip install -e .[molecules]
+          python -m pip install notebook
+      - name: Backup Reference Splits
+        run: |
+          cd examples
+          cp QM9_splits/QM9_splits_kmeans.pkl REFERENCE_QM9_splits_kmeans.pkl
```

Comment:
As a super minor question, should these technically be `mv` rather than `cp`? The original files are overwritten later when running the `*_make_splits` jupyter script so `cp` should be fine unless something goes wrong and they're somehow not overwritten. Feel free to let me know if I'm being overly paranoid haha

Reply:
Good point, I think this will help with clarity. Added.